modMcf/src/org/aion/mcf/trie/TrieImpl.java

```diff
@@ -2,16 +2,16 @@
 
 import static java.util.Arrays.copyOfRange;
 import static org.aion.base.util.ByteArrayWrapper.wrap;
-import static org.aion.base.util.ByteUtil.EMPTY_BYTE_ARRAY;
-import static org.aion.base.util.ByteUtil.matchingNibbleLength;
 import static org.aion.crypto.HashUtil.EMPTY_TRIE_HASH;
 import static org.aion.rlp.CompactEncoder.binToNibbles;
 import static org.aion.rlp.CompactEncoder.hasTerminator;
 import static org.aion.rlp.CompactEncoder.packNibbles;
 import static org.aion.rlp.CompactEncoder.unpackToNibbles;
 import static org.aion.rlp.RLP.calcElementPrefixSize;
+import static org.aion.util.bytes.ByteUtil.matchingNibbleLength;
```

Comment:
i think it's supposed to be called Nybble

Reply:
Since this method name predates the current code changes, I will not update it here. Perhaps it can be done as part of the larger upcoming refactoring. The `ByteUtil` class will likely need to be moved to `modUtil` so the change can be applied at that point.